job_runner/reporting/misc.py

```diff
@@ -0,0 +1,8 @@
+from abc import ABC, abstractmethod
+
+
+class Report(ABC):
```

Comment:
`reporting` is a gerund, and we refer to these `File` `Network` and `Event` as reporters, so maybe just call it a `Reporter`?